code/modules/meteors/meteors.dm

```diff
@@ -510,5 +512,45 @@ GLOBAL_LIST_INIT(meteorsSPOOKY, list(/obj/effect/meteor/pumpkin))
 	. = ..()
 	meteorsound = pick('sound/hallucinations/im_here1.ogg','sound/hallucinations/im_here2.ogg')
 //////////////////////////
+
+/obj/effect/meteor/gaseous
+	name = "gas filled meteor"
+	dropamt = 1
+	threat = 10
+	hits = 10
+
+/obj/effect/meteor/gaseous/meteor_effect()
+	. = ..()
+
+	var/static/list/gases_to_spawn = list(
+		/datum/gas/nitrogen,
+		/datum/gas/oxygen,
+		/datum/gas/carbon_dioxide,
+		/datum/gas/plasma,
+		/datum/gas/nitrous_oxide,
+		/datum/gas/healium,
+		/datum/gas/freon,
+		/datum/gas/tritium,
+		/datum/gas/miasma,
+		/datum/gas/hydrogen,
+		/datum/gas/water_vapor,
+		/datum/gas/hypernoblium
+		)
```

Comment:
gasses that do not cause an instant bad reaction or a fire will have zero effect, you want to have a tritfire/plasmafire meteor and not a co2 meteor if you wanna actually make an impact

Reply:
other mixes are welcome other than the standard fires